tests/OSPSuite.Core.IntegrationTests/ModelConstructorIntegrationTests.cs

```diff
@@ -563,7 +564,7 @@ protected override void Context()
       [Observation]
       public void should_be_able_to_create_the_model()
       {
-         _result.ValidationResult.ValidationState.ShouldBeEqualTo(ValidationState.Valid);
+         _result.ValidationResult.ValidationState.ShouldNotBeEqualTo(ValidationState.Invalid);
```

Comment:
This tests that the warning was created since our use case actually has a parameter defined in a container that does not exist